src/org/opendatakit/briefcase/ui/export/ExportPanelForm.form

```diff
@@ -57,6 +51,14 @@
             <properties/>
             <border type="none"/>
             <children>
+              <component id="b7dc5" class="javax.swing.JLabel" binding="exportingLabel">
+                <constraints/>
+                <properties>
+                  <minimumSize width="86" height="21"/>
+                  <preferredSize width="86" height="21"/>
+                  <text value="Exporting.........."/>
```

Comment:
So, I've tested this and you should give it at least 110 width... Otherwise it doesn't fit.

I'd like to suggest using "Exporting..." (three dots) and a width of 80 to make it compact.

This also is missing a `<visible value="false"/>` to make it not visible when you first visit the tab.